core/server/api/utils.js

```diff
@@ -194,9 +194,10 @@ utils = {
      * ## Handle Permissions
      * @param {String} docName
      * @param {String} method (browse || read || edit || add || destroy)
+     * * @param {Array} unsafeAttrNames - attribute names (e.g. post.status) that could change the outcome
      * @returns {Function}
      */
-    handlePermissions: function handlePermissions(docName, method) {
+    handlePermissions: function handlePermissions(docName, method, unsafeAttrNames) {
```

Comment:
I wonder why we have to define the unsafe attributes on the API level?
Isn't that something the model permissible function can control alone?
The permissible function just needs access to the data which was transmitted.

Reply:
I agree long term it probably makes more sense to pass all the data around, but short term I wanted to make it absolutely explicit which attributes could have permissions based upon them.

The reason is quite "soft" - I want to make it difficult to accidentally hard code attribute-based rules right now. This is so we can keep track, and hopefully come up with a better way to represent these rules, e.g. in the database or with a different structure like JSON policies.

Reply:
Ok 👍